### PR TITLE
Configure time position

### PR DESCRIPTION
**Pull request by lotsabackscatter**
Add a **timePosition** calendar attribute
Options: side, default or hidden

This new attribute allows configuring where the time should be positioned on the 'day view'.

**side**: on the side, outside of the calendar
**hidden**: kept out of sight; concealed ;)
**default**: on the inside of the calendar (any missing or unknown value will be interpreted as this option)

### CODE REVIEW

**Comment by lotsabackscatter** [comment]
Ah yep, I wondered about the dist stuff.
I'll sort it out.

File: src/directives/mwlCalendarDay.js [hunk 1]
```diff
@@ -105,7 +109,8 @@ angular
         dayViewEventWidth: '=',
         customTemplateUrls: '=?',
         cellModifier: '=',
-        templateScope: '='
+        templateScope: '=',
+        timePosition: '='
```
Comment (mattlewis92) [inline src/directives/mwlCalendarDay.js]: Travis is failing because of a trailing space at the end of this line, can you fix it please?

**Comment by mattlewis92** [comment]
The dist files are still changed, I think if you [sync your fork](https://help.github.com/articles/syncing-a-fork/) with this repos master and then just copy the dist files across it should fix it.

**Comment by lotsabackscatter** [comment]
How's that @mattlewis92?

**Comment by mattlewis92** [comment]
The dist files are still included, can you fix them? (Maybe just download them from master and overwrite manually)

File: src/directives/mwlCalendarDay.js [hunk 2]
```diff
@@ -8,6 +8,10 @@ angular
 
     var vm = this;
 
+    vm.timePosition = vm.timePosition ? vm.timePosition : 'default';
+    vm.timeHidden = vm.timePosition === 'hidden';
+    vm.timePositionOffset = vm.timePosition !== 'default' ? 0 : 60;
```
Comment (mattlewis92) [inline src/directives/mwlCalendarDay.js]: In order for this to work it needs to be inside the `refreshView()` function, as with ng 1.6+ these inputs wont be initialised on the components construction.

**Comment by mattlewis92** [comment]
Tried to fix the dist files myself through githubs UI but it keeps adding new lines to the end of the files

**Comment by lotsabackscatter** [comment]
Thought I'd reverted the dist... I'll have to take another look.

I'll take a look at that `refreshView()` issue. Haven't upgraded to 1.6 yet.

**Comment by mattlewis92** [comment]
I think it's because the git history on the dist files changed, there's probably a fancy way of fixing it via git but a quick and dirty fix would just be to download the dist files from the master repo and copy them into your branch, then git should detect that there are no changes.

If you sync from the master branch where angular has been updated to 1.6, you'll see your demo doesn't work properly due to the upgrade.

**Comment by mattlewis92** [comment]
Released as 0.28.0, thanks for the contribution! :octocat:

**Pull request merged**